pkg/network/common/dns.go

```diff
@@ -189,3 +233,55 @@ func ipsEqual(oldips, newips []net.IP) bool {
 	}
 	return true
 }
+
+// getDNSResolvers extracts nameservers from the given resolver config file
+func getDNSResolvers(resolverConfigFile string) ([]string, error) {
```

Comment:
miekg/dns should already have a parser for this... might be better not to write your own.